### DISCUSSION TITLE
Pico Theme Development Checklist:

### DISCUSSION THREAD

**Discussion by ratulkhan74** [cadentcom/pico-wp-block-theme#7] [General]
**Completed Tasks:**

Theme Settings to Create a Block Theme
Home Page

**To be Completed:**

Blog Page
Single-Post Page
Category-Post Page
Post Comments
Post Search form
Post Search Page
404 Page
Archive posts

Note: We have a theme, but we are not ready to submit it to wordpress.org. We need to complete the checklist first, and, most importantly, while we have a theme skeleton, we lack a visual design (face). Therefore, we require a proper design to finalize the theme.
Please check the URL: https://wordpress.org/themes/tags/full-site-editing/
This link is the wordpress.org theme page link. Upon reviewing the page, you'll notice that each theme has a distinct design in its specific category. Thus, it is imperative for us to have a proper design to achieve our target

**Comment by stratofax** [comment]
Thank you for the detailed checklist and next steps. I'll send you a design separately.

**Comment by stratofax** [comment]
Also, @ratulkhan74  can you let me know what's on each branch? There are currently 4 branches, besides main:

- dev
- demo
- demo2 (this is the only branch with an open pull request)
- block theme

Please let me know which branch(es) you plan to use for development. I'd like to designate a branch that consists of working code, like a "nightly build. Please let me know which one you recommend, or if you'd like to create a new branch for this purpose. 

Can we remove any of the branches? I'd like to create a new discussion to document each active, long-lasting branch, as well.

**Reply by ratulkhan74** [reply]
@stratofax I have created another discussation though,

I have used these branches for our pico theme development

- dev
- demo
- demo2

for our new theme i want to start with a new branch and we can remove **_block theme branch_**  we can remove the demo branch also

**Reply by stratofax** [reply]
This discussion about branches is now at #9 -- please post new comments there!
